web-api/src/persistence/postgres/cases/getEligibleCasesForTrialCity.ts

```diff
@@ -0,0 +1,57 @@
+import { CASE_STATUS_TYPES } from '@shared/business/entities/EntityConstants';
+import { getDbReader } from '@web-api/database';
+import { getCasesByDocketNumbers } from '@web-api/persistence/postgres/cases/getCasesByDocketNumbers';
+import { Kysely } from 'kysely';
+import { Database } from '@web-api/database-schema';
+
+export const eligibleCasesQuery = ({
```

Comment:
I'm not sure exactly where to put this comment, but I think we may not be properly hiding private data. See below, where I sealed the address: I can't see the address as a petitions clerk on the main case page, but can see it in the network response on the trial session planning location screen.
<img width="1491" alt="Screenshot 2025-05-23 at 9 54 49 AM" src="https://github.com/user-attachments/assets/29d426a1-8b44-46b8-ac62-8c5c65f8f11d" />
<img width="893" alt="Screenshot 2025-05-23 at 9 54 21 AM" src="https://github.com/user-attachments/assets/3b7d26b2-7f0b-4ce5-ba99-badc224e7a28" />

Reply:
Good catch, this will be a problem anywhere we use `getCasesByDocketNumbers` and don't use `CaseFactory`

Reply:
Or another filtering method

Reply:
100%. CaseFactory had its heart in the right place, but as I've worked on 10502 more, I've realized it doesn't really solve the problem. Its main benefit at this point is just clarifying the logic for how to clean the data.